axelrod/strategy_transformers.py

```diff
@@ -309,9 +350,46 @@ def mixed_wrapper(player, opponent, action, probability, m_player):
 
     return action
 
+
 MixedTransformer = StrategyTransformerFactory(
     mixed_wrapper, name_prefix="Mutated")
 
+
+def joss_ann_wrapper(player, opponent, proposed_action, probability):
+    """Wraps the players strategy function to produce the Joss-Ann.
```

Comment:
Can we include the bibliographic reference too as well (so with `[Ashlock...]_` and the relevant info in the bibliography.rst file.